### DISCUSSION TITLE
Force to update dns

### DISCUSSION THREAD

**Discussion by lukasa1993** [Kong/kong#7449] [Help] [answered]
is there any way maybe from admin api to force update services ip ? it seems to be caching it and when i update docker swarm service it takes few minutes to update usually less then 5 but still i would like to update it manually

**Comment by Tieske** [comment, marked as answer]
No. Kong honors the DNS records TTL. There is a setting to override TTL's if you set that to something short, it could alleviate the problem maybe. Otherwise you would have to execute a "reload" to clear the DNS cache.

**Comment by lukasa1993** [comment]
```
      - KONG_DNS_STALE_TTL=120      
      - KONG_DNS_ORDER=A      
      - KONG_DNS_NOT_FOUND_TTL=1      
      - KONG_DNS_NO_SYNC=on
 ```
this is what i use what is "reload" ? and how to execute it ?

**Comment by Tieske** [comment]
reload: https://docs.konghq.com/gateway-oss/2.4.x/cli/#kong-reload
ttl override: https://docs.konghq.com/gateway-oss/2.4.x/configuration/#dns_valid_ttl
